File: 0162-find-peak-element/0162-find-peak-element.py
class Solution:
    def findPeakElement(self, nums: list[int]) -> int:
        peakind = 0
        max = float(-inf)
        for i in range(0, len(nums)):
            if (nums[i]>max):
                max = nums[i]
                peakint = i
        return peakint
        

File: 0162-find-peak-element/0162-find-peak-element.java
class Solution 
{
    public int findPeakElement(int[] nums) 
    {
        int peakind = 0;
        int max = Integer.MIN_VALUE;
        for(int i =0;i<nums.length;i++)
        {
            if(nums[i]>max)
            {
                max = nums[i];
                peakind = i;
            }
        }
        return peakind;
    }
}